ui/components/ui/site-origin/index.scss

```
@@ -0,0 +1,5 @@
+.site-origin {
+  .chip__left-icon {
+    padding: 12px 0 12px 12px;
```

Suggest using 8px grid
```suggestion
    padding: 8px 0 8px 8px;
```

---

This would be a bit closer to the designs:

```suggestion
    padding: 4px 0 4px 8px;
```

Currently it looks far too tall.